EosioSwift.xcodeproj/xcshareddata/xcschemes/EosioSwift.xcscheme

```diff
@@ -22,7 +22,7 @@
          </BuildActionEntry>
          <BuildActionEntry
             buildForTesting = "YES"
-            buildForRunning = "NO"
+            buildForRunning = "YES"
```

Comment:
This can really raise compile times for development but since we're not under heavy active dev any more it should be fine.  Some of the other repos may have build for run turned off as well.

Reply:
This is resolved. It was a different issue I enabled this for but now all those changes are rolled back.